dc-commons-xml/src/main/java/de/digitalcollections/commons/xml/xpath/XPathRoot.java

```diff
@@ -0,0 +1,32 @@
+package de.digitalcollections.commons.xml.xpath;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation to set the default namespace and path prefixes for all
+ * subsequent {@link XPathBinding} annotations.
+ */
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface XPathRoot {
+
+  /**
+   * @return the default namespace, e.g. <code>http://www.tei-c.org/ns/1.0"</code> (optional)
+   */
+  String defaultNamespace() default "";
+
+  /**
+   * @return an array of path prefixes, e.g. <code>/tei:TEI/tei:teiHeader/tei:fileDesc/tei
```

Comment:
Das ist a bissle lang für nur `@return`, magst das in einen Absatz oben drüber packen?

Reply:
done